test/unit/grammar/range_rule.cpp

```diff
@@ -98,18 +98,6 @@ struct range_rule_test
             BOOST_TEST_EQ(v3.size(), 0);
         }
 
-        // range(range&&)
```

Comment:
These removed tests broke?

Reply:
I figure there's no need for move members since copies are cheap.